src/harness/util.py

```diff
@@ -408,3 +408,15 @@ def filterChannelsByFrequencyRange(channels, freq_range):
       and
       channel['frequencyRange']['highFrequency'] <= freq_range['highFrequency']
   ]
+def countdown(time_to_wait):
```

Comment:
Just use sleep -- there's no need to reinvent the wheel here.

Reply:
As per   suggestion used sleep